go/common/logging/multi.go

```diff
@@ -0,0 +1,39 @@
+package logging
+
+import (
+	"github.com/go-kit/log"
+)
+
+type multiLogger struct {
+	loggers []log.Logger
+}
+
+func newMultiLogger(loggers []log.Logger) *multiLogger {
+	return &multiLogger{
+		loggers: loggers,
+	}
+}
+
+func (l *multiLogger) Log(keyvals ...any) error {
+	for _, logger := range l.loggers {
+		if logger == nil {
```

Comment:
Is there a use case for this being nil, or would it be reasonable to expect that this should not be nil? (unless a developer error), and should we emit this check?

Reply:
Changed.